workers/generic-worker/build.sh

```diff
@@ -81,7 +93,7 @@ else
   esac
 fi
 
-find "${GOPATH}/bin" -name 'generic-worker*'
+find $OUTPUT_DIR -name 'generic-worker*'
```

Comment:
There should be double quotes around `$OUTPUT_DIR` in case the checkout directory contains spaces (which it could if developers specify an output directory).